kolibri/deployment/default/urls.py

```diff
@@ -49,6 +49,10 @@
     ] + urlpatterns
 
 if getattr(settings, 'REST_SWAGGER', False):
+    from rest_framework_swagger.views import get_swagger_view
+
+    schema_view = get_swagger_view(title='Kolibri API')
+
```

Comment:
This needs an `else`, otherwise `schema_view` is undefined, no?

Reply:
It's only used inside the if statement though.

Reply:
This was a pretty useless Saturday evening review all in all.